datadog/resource_datadog_screenboard.go

```diff
@@ -88,6 +88,115 @@ func resourceDatadogScreenboard() *schema.Resource {
 		},
 	}
 
+	tileDefApmOrLogQuery := &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		MaxItems: 1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"index": {
+					Type:     schema.TypeString,
+					Required: true,
+				},
+				"compute": &schema.Schema{
+					Type:     schema.TypeMap,
+					Required: true,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
+							"aggregation": {
+								Type:     schema.TypeString,
+								Required: true,
+							},
+							"facet": {
+								Type:     schema.TypeString,
+								Optional: true,
+							},
+							"interval": {
+								Type:     schema.TypeInt,
```

Comment:
This looks wrong for the legacy API. I think it should be a string. Try to import a dashboard created by the UI. You will get:
`Error: datadog_screenboard.import (import id:xxx-xxx-xxx): import datadog_screenboard.import (id: xxx-xxx-xxx): json: cannot unmarshal string into Go struct field TileDefApmOrLogQueryCompute.interval of type int